cnab-config/castai-agent/values.yaml

```diff
@@ -0,0 +1,142 @@
+# Default values for castai-agent.
```

Comment:
do we need to clone the whole config?

can we merge original values.yaml and wanted updates for the cnab/values version?

Reply:
Will look into this

Reply:
As suggested, configured CNAB script to merge it's values to original values.yaml file